sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/src/BlobPartitionManager.cs

```diff
@@ -49,17 +49,19 @@ public sealed class BlobPartitionManager : PartitionManager
         ///   Retrieves a complete ownership list from the storage blob service.
         /// </summary>
         ///
+        /// <param name="fullyQualifiedNamespace">The fully qualified host name for the Event Hubs namespace the ownership are associated with.</param>
```

Comment:
It might be helpful for the user if we provide an example of `fullyQualifiedNamespace` in the docs. Something like:
```
<param name="fullyQualifiedNamespace">The fully qualified Event Hubs namespace. This is likely to be similar to
<yournamespace>.servicebus.windows.net</param>
```

Reply:
+1 on that.   :)